packages/core-components/report.api.md

```diff
@@ -1411,7 +1411,7 @@ export type TableFilter = {
 // Warning: (ae-missing-release-tag) "TableFiltersClassKey" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export type TableFiltersClassKey = 'root' | 'value' | 'heder' | 'filters';
+export type TableFiltersClassKey = 'root' | 'value' | 'header' | 'filters';
```

Comment:
I'll need confirmation from @Rugvip but I don't think we can actually do this. There's some issue around how this works where we can't add new keys or changes the existing ones. Hopefully I'm wrong and just remembering things incorrectly.

Reply:
You're probably right! My understanding  is that if 2 versions of a plugin are installed, there is a type conflict for the MUI theme overrides. When running `tsc:full`, you get an error that can't be overridden easily as it's in third-party code.
This is why the `resolutions` in the main package.json was required, to fix this type error in the repo.

It could be annoying for consumers, so I fully understand if we rollback and add a comment instead.